Dashboard Import is broken in master

<!--
I tried the latest nightly from master - https://grafana.com/grafana/download/6.6.0-39047d94pre?platform=windows
-->

**What happened**:
Dashboard import does nothing. Click on import, and see nothing happens.
**What you expected to happen**:
Able to import dashboard
**How to reproduce it (as minimally and precisely as possible)**:
Install - https://grafana.com/grafana/download/6.6.0-39047d94pre?platform=windows
Login & try importing an dashboard.
**Anything else we need to know?**:
Browser debug console has the following error -
store updating angular $location.url /?orgId=1
:3000/?orgId=1:1 Uncaught (in promise) undefined
filters.ts:10 store updating angular $location.url /datasources/edit/2?gettingstarted
angular.js:14800 Error: [$injector:unpr] Unknown provider: eProvider <- e <- validationSrv
http://errors.angularjs.org/1.6.9/$injector/unpr?p0=eProvider%20%3C-%20e%20%3C-%20validationSrv
    at angular.js:116
    at angular.js:4887
    at Object.s [as get] (angular.js:5047)
    at angular.js:4892
    at s (angular.js:5047)
    at u (angular.js:5072)
    at Object.instantiate (angular.js:5118)
    at Object.<anonymous> (angular.js:4955)
    at Object.invoke (angular.js:5106)
    at Object.$get (angular.js:4939) "<div ng-view="">"
(anonymous) @ angular.js:14800
filters.ts:10 store updating angular $location.url /dashboard/new?orgId=1
angular.js:14800 Error: [$injector:unpr] Unknown provider: eProvider <- e <- validationSrv
http://errors.angularjs.org/1.6.9/$injector/unpr?p0=eProvider%20%3C-%20e%20%3C-%20validationSrv
    at angular.js:116
    at angular.js:4887
    at Object.s [as get] (angular.js:5047)
    at angular.js:4892
    at s (angular.js:5047)
    at u (angular.js:5072)
    at Object.instantiate (angular.js:5118)
    at Object.<anonymous> (angular.js:4955)
    at Object.invoke (angular.js:5106)
    at Object.$get (angular.js:4939) "<div ng-view="">"
(anonymous) @ angular.js:14800


**Environment**:
- Grafana version: nightly (https://grafana.com/grafana/download/6.6.0-39047d94pre?platform=windows)
- Data source type & version:
- OS Grafana is installed on: windows 10
- User OS & Browser: chrome Version 79.0.3945.79 (Official Build) (64-bit)
- Grafana plugins:
- Others:

--- a/public/app/features/manage-dashboards/services/ValidationSrv.ts
+++ b/public/app/features/manage-dashboards/services/ValidationSrv.ts
@@ -9,6 +9,7 @@ const hitTypes = {
 export class ValidationSrv {
   rootName = 'general';
 
+  /** @ngInject */
   constructor(private backendSrv: BackendSrv) {}
 
   validateNewDashboardName(folderId: any, name: string) {

